loom/graphics/gfxGraphics.h

```diff
@@ -163,6 +165,23 @@ namespace GFX
 #undef GFX_PROC_VOID
     } GL_Context;
 
+// Represents graphics view frame properties that can change from frame buffer to frame buffer
+typedef struct GraphicsFrame {

```

Comment:
Given we render "frames" maybe it would be better to call this GraphicsRenderTarget?